textile/chat-features.textile

```diff
@@ -1567,52 +1591,102 @@ h3(#chat-structs-occupancy-data). Occupancy Data
   }
 </pre>
 
+h2(#common-error-codes). Commmon Error Codes used by Chat
+
+This section contains error codes that are common across Ably, but the Chat SDK makes use of.
+
+<pre>
+    // The request was invalid.
+    // To be accompanied by status code 400.
+    BadRequest = 40000,
+
+    // Invalid argument provided.
+    // To be accompanied by status code 400.
+    InvalidArgument = 40003,
+
+    // Invalid client ID.
+    // To be accompanied by status code 400.
+    InvalidClientId = 40012,
+
+    // Resource has been disposed.
+    // To be accompanied by status code 400.
+    ResourceDisposed = 40014,
```

Comment:
Can we update CHA-RC1f8 to refer to this code?

Reply:
Yes